specifyweb/frontend/js_src/lib/components/WorkBench/WbView.tsx

```diff
@@ -64,32 +48,36 @@ export type Workbench = {
   disambiguation: Disambiguation;
   validation: WbValidation;
   data: RA<RA<string | null>>;
-  hot: Handsontable;
+  hot: Handsontable | undefined;
```

Comment:
(optional) if you can delay creation of the `Workbench` object until `hot` is no longer undefined, that would simplify your life